higra/attribute/tree_attributes.py

```diff
@@ -675,3 +675,43 @@ def attribute_topological_height(tree):
                                            hg.Accumulators.max)
 
     return res
+
+
+
+@hg.argument_helper(hg.CptHierarchy)
+@hg.auto_cache
+def attribute_moment_of_inertia(tree, altitudes, leaf_graph):
+    """
+    
+    Moment of inertia (first Hu moment) of each node of the given tree.   
```

Comment:
Please, clearly state that the function only works if ``leaf_graph`` is a 2d grid graph. It would also be nice to recall the definition of the 2d moment of inertia used here.